backend/src/parser/TheParser.ts

```diff
@@ -345,6 +346,15 @@ export default class TheParser {
         return { ...result, text };
     }
 
+    parseCut(node: Element): ParseResult {
+        const title = node.attribs['title'] || 'Тони, открой собаку';
+
+        const result = this.parseChildNodes(node.children);
+        const text = `<details class="cut"><summary>${htmlEscape(title)}</summary>${result.text}<div role="button"></div></details>`;
+
+        return { ...result, text } ;
```

Comment:
Need to propagate all other parts of the `result` as well (mentions, links, etc).